engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/subprocess/NonInterruptingEventSubprocessTest.java

```diff
@@ -69,66 +68,80 @@ public class NonInterruptingEventSubprocessTest {
   @Parameterized.Parameters(name = "{0} event subprocess")
   public static Object[][] parameters() {
     return new Object[][] {
+      //      {
```

Comment:
🔧  Don't forget to uncomment those lines 😃

Reply:
😅 Oops!  Thanks.